ethsigner/commandline/src/main/java/tech/pegasys/ethsigner/CommandlineParser.java

```diff
@@ -60,6 +60,7 @@ public boolean parseCommandLine(final String... args) {
 
     try {
       commandLine.parseWithHandlers(new RunLast().useOut(output), new ExceptionHandler<>(), args);
+      baseCommand.validateOptions(commandLine, LOG);
```

Comment:
I am slightly concerned that this "validateOptions" might not be part of "business logic" ... specially after the upgrade "execute" method coming in from other PR. In other words, standard parsing may have been successful already which would have caused http server to be up. i.e. The options validation (and exception raising) should happen in or around `tech/pegasys/ethsigner/SignerSubCommand.java:35` or tech/pegasys/ethsigner/core/EthSigner.java:52

Reply:
fixed